docs/hackathon/startup-guide.md

```diff
@@ -1,64 +1,70 @@
 
-```bash
-git clone https://github.com/nearprotocol/nearcore.git
-cd nearcore
-./scripts/start_testnet.py
-```
-
-This will ask you for account id to run validator on \(if you want to just run a node, you can pass empty\). Will print the public key that should be used for staking. And will start node in background inside the docker.
+## Sounds amazing, how do I get started?
 
-If you want to check the logs inside the docker, you can use `docker logs --follow nearcore`.
+The very first thing that you will have to do is set-up your wallet by creating a [NEAR Account](../local-setup/create-account). Once you have done this, please come back to this page and we will continue.
 
-Alternatively, you can build and run validator on this machine, just use `--nodocker` flag to switch off the Docker. This will install Rust and compile the binary.
+There is no "right" way of starting development on NEAR. Depending on your goal, you can start differently. We have outlined some of the options and steps below.
 
-## Running official TestNet on GCP
+### Quick Start Development
 
-Create new instance, with at least:
+You want to check-out some applications built on NEAR, have a look at smart contracts, and run a dApp (decentralized Application) in no time. 
 
-* 2 vCPU and 3.75 GB of RAM.
-* Select Ubuntu 18.04 LTS or later.
-* Allocate 100GB of persistent storage.
+1. Open the [Near Studio IDE](https://near.dev/). There you will have a selection of 4 projects to check out. 
+2. To learn more about our Studio IDE and smart contracts, please head over to the Quick Start Section.
+    * [Smart Contracts](../quick-start/smart-contracts.md);
+    * [Near Studio IDE](../quick-start/near-studio-ide.md);
+    * Lastly, have a look at the file structure in the [NEAR Development Overview](../quick-start/development-overview).
 
-Add firewall rules to allow traffic to 24567 port from all IPs \(0.0.0.0/0\)
+When you are using Near Studio, the IDE will handle deploying your smart contracts to the NEAR blockchain. 
 
-SSH into the machine \(there is "SSH" button in the console or use gcloud ssh command\).
+For the start of the hackathon, it's fine to do development in the studio, but eventually you're going to want to set up a GitHub repo and share code with your team. That will be easier to do with a local setup.
 
-Run:
+### Setup a local environment 
+Let's set up your development environment. 
 
-```bash
-sudo apt update
-sudo apt install -y git binutils-dev libcurl4-openssl-dev zlib1g-dev libdw-dev libiberty-dev cmake gcc g++ python docker.io protobuf-compiler
-```
+1. First, you want to install the [near shell](../development/near-clitool) in your terminal;
```

Comment:
can we call it `NEAR Shell`?  in general, i'd like to start using `NEAR [A-Z]___` pattern for user facing systems like NEAR Wallet, NEAR Explorer, NEAR Studio, etc.

Reply:
Good point; we have not been consistent with the naming so far